utilities/EluvioLiveCli.js

```diff
@@ -492,6 +498,21 @@ yargs(hideBin(process.argv))
 
 		   })
 
+	.command('shuffle <file> [options]',
+		'Sort each line deterministically based on the seed', (yargs) => {
+			yargs
+				.positional('file', {
+					describe: 'File path',
+					type: 'string'
+				})
+				.option('seed', {
+					describe: 'Determines the order',
```

Comment:
Be good to explain what happens if no seed (for example uses empty string if not specififed)